spec/requests/submissions_spec.rb

```diff
@@ -111,4 +122,103 @@
       end
     end
   end
+
+  describe "PATCH /submissions/:id" do
+    context "when logged in as a challenge manager" do
+      let(:user) { create_user(role: "challenge_manager") }
+      let(:submission) { create(:submission, challenge: challenge, phase: phase) }
+      let(:evaluator) { create(:user, role: 'evaluator') }
+
+      before do
+        ChallengeManager.create(user: user, challenge: challenge)
+      end
+
+      context "updating judging status" do
+        it "updates eligibility status to selected" do
+          patch submission_path(submission), params: {
+            submission: { judging_status: 'selected' }
+          }, as: :json
+
+          expect(response).to have_http_status(:ok)
+          expect(submission.reload.judging_status).to eq('selected')
+        end
+
+        it "updates advancement status to winner when evaluations are complete" do
+          submission.update!(judging_status: 'selected')
+          create(:evaluator_submission_assignment,
+                 submission: submission,
+                 evaluator: evaluator,
+                 status: :assigned,
+                 evaluation: create(:evaluation, completed_at: Time.current))
+
+          patch submission_path(submission), params: {
+            submission: { judging_status: 'winner' }
+          }, as: :json
+
+          expect(response).to have_http_status(:ok)
+          expect(submission.reload.judging_status).to eq('winner')
+        end
+
+        it "prevents advancement when evaluations are not complete" do
+          submission.update!(judging_status: 'selected')
+          create(:evaluator_submission_assignment,
+                 submission: submission,
+                 evaluator: evaluator,
+                 status: :assigned)
+
+          patch submission_path(submission), params: {
+            submission: { judging_status: 'winner' }
+          }, as: :json
+
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(submission.reload.judging_status).to eq('selected')
+        end
+
+        it "prevents eligibility changes when evaluators are assigned" do
+          create(:evaluator_submission_assignment,
+                 submission: submission,
+                 evaluator: evaluator,
+                 status: :assigned)
+
+          patch submission_path(submission), params: {
+            submission: { judging_status: 'selected' }
+          }, as: :json
+
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(submission.reload.judging_status).to eq('not_selected')
+        end
+
+        it "prevents changing judging_status from 'not_selected' to 'selected' when eligibility checkbox is disabled" do
```

Comment:
can we reverse the direction here and test that it
```suggestion
        it "prevents changing judging_status from 'selected' to 'not_selected' when eligibility checkbox is disabled" do
```

Reply:
(i think this and the previous spec are testing the same thing basically?)